includes/class-wc-subscriptions-email-notifications.php

```diff
@@ -297,4 +300,53 @@ public static function add_settings( $settings ) {
 		WC_Subscriptions_Admin::insert_setting_after( $settings, WC_Subscriptions_Admin::$option_prefix . '_miscellaneous', $notification_settings, 'multiple_settings', 'sectionend' );
 		return $settings;
 	}
+
+	/**
+	 * Maybe add an admin notice to inform the store manager about the existance of the notifications feature.
+	 */
+	public static function maybe_add_admin_notice() {
+
+		// If the notifications feature is enabled, don't show the notice.
+		if ( self::notifications_globally_enabled() ) {
+			return;
+		}
+
+		$option_name = 'wcs_hide_customer_notifications_notice';
+		$nonce       = '_wcsnonce';
+		$action      = 'wcs_hide_customer_notifications_notice_action';
+
+		// First, check if the notice is being dismissed.
+		$nonce_argument = sanitize_text_field( wp_unslash( $_GET[ $nonce ] ?? '' ) );
+		if ( isset( $_GET[ $action ], $nonce_argument ) && wp_verify_nonce( $nonce_argument, $action ) ) {
+			update_option( $option_name, 'yes' );
+			wp_safe_redirect( remove_query_arg( [ $action, $nonce ] ) );
+			return;
+		}
+
+		if ( 'yes' === get_option( $option_name ) ) {
+			return;
+		}
+
+		$admin_notice = new WCS_Admin_Notice( 'notice' );
+		$admin_notice->set_simple_content(
+			esc_html__(
+				'New customer email reminders for renewals, expirations, and free trials are now available! Enable and configure these features in WooCommerce > Settings > Subscriptions to control when your customers receive important updates.',
```

Comment:
I think the wording can be a bit confusing. We call it reminders here, but down in the settings the section is called notifications... But let's iterate on this once James is back.